### DISCUSSION TITLE
Welcome to brainrot.js Discussions!

### DISCUSSION THREAD

**Discussion by noahgsolomon** [noahgsolomon/brainrot.js#12] [Announcements]
Ask a question or suggest improvements.

**Comment by failureincode** [comment]
it gives me an error when im trying to install npm npm : The term 'npm' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, 
verify that the path is correct and try again.
At line:1 char:1
+ npm install
+ ~~~
    + CategoryInfo          : ObjectNotFound: (npm:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Reply by noahgsolomon** [reply]
you prob don't have node installed on your device. i think npm comes with node but make sure of that.
